app/src/test/java/io/crate/node/NodeSettingsTest.java

```diff
@@ -138,13 +140,7 @@ public void testDefaultPaths() {
 
     @Test
     public void testDefaultPorts() {
-        assertEquals(
-            Constants.HTTP_PORT_RANGE,
-            node.settings().get("http.port")
-        );
-        assertEquals(
-            Constants.TRANSPORT_PORT_RANGE,
-            node.settings().get("transport.tcp.port")
-        );
+        assertThat(HttpTransportSettings.SETTING_HTTP_PORT.getDefault(node.settings()).getPortRangeString(), is("4200-4300"));
+        assertThat(node.settings().get("transport.tcp.port"), nullValue());
```

Comment:
sry I did not mention it explicitly, but this should also be adjusted. as said I'd remove the complete test instead.